modules/client/src/test/java/org/apache/ignite/client/fakes/FakeInternalTable.java

```diff
@@ -524,9 +521,9 @@ public ScheduledExecutorService streamerFlushExecutor() {
     }
 
     @Override
-    public CompletableFuture<ClusterNode> partitionLocation(ReplicationGroupId replicationGroupId) {
+    public CompletableFuture<ClusterNode> partitionLocation(int partitionId) {
```

Comment:
Here and there let's name it as partitionIndex in order not to mess with groupId that is either TablePartitionId or ZonePartitionId.

Reply:
:ok_hand: